const makeOrder= async(req,res)=>{
    try {
        
    } catch (error) {
        return res.status(200).json({message:"Internal server error",error:error.message})
    }
}